tests/functional/object/mcg/test_noobaa_bucket_error_state_mode.py

```
@@ -0,0 +1,462 @@
+
+            prometheus_api = PrometheusAPI(threading_lock=threading_lock)
+            alert_name = "NooBaaBucketErrorState"
+            alerts = wait_for_alert_firing(
+                api=prometheus_api,
+                alert_name=alert_name,
+                timeout=600,
+                expected_message_substr="EXCEEDING_QUOTA",
+            )
+
+            alert_labels = alerts[0]["labels"]
+            logger.info(f"Alert {alert_name} fired with labels: {alert_labels}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)